### ISSUE TITLE
Accessibility: [Screen Reader]: remove unwanted aria roles such as aria-label

### ISSUE THREAD

**Issue by ericmaino** [microsoft/BotFramework-WebChat#3090]
![image](https://user-images.githubusercontent.com/1005609/78629201-32e57400-784b-11ea-8a47-f5b07992fda4.png)

**Actual**
aria-label is un-necessarily added in markup

**Expected**
As inner text highlighted in “green” is present, aria-label is not needed.

**WCAG Violation**
Parsing 4.1.1
https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html

**Comment by ericmaino** [comment]
@corinagum Yes the photos were against v3, however this issue is still present in v4. Please re-open this bug for tracking and tag with Accessibility

**Comment by compulim** [comment]
This is related to Adaptive Cards.

The content author is creating the bubble content (with 3 buttons) using hero card, which is then rendered by the Adaptive Card library. As you see, the `class` are prefixed with `ac-`, it means Adaptive Cards.

Please contact the Adaptive Card team for this issue.

**Comment by compulim** [comment]
I have filed bug https://github.com/microsoft/AdaptiveCards/issues/3916 to Adaptive Cards team. Please track there.
